src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.Figure;
+import core.basesyntax.randomizers.FigureSupplier;
+
+public class Application {
+    public static void main(String[] args) {
+        FigureSupplier figure = new FigureSupplier();
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i < figures.length; i++) {
+            if (i < figures.length / 2) {
+                figures[i] = figure.getRandomFigure();
+                System.out.println(figures[i].drawFigure());

```

Comment:
`System.out.println(figures[i].drawFigure());`
move this line below if-else

Reply:
Done!